src/v/cloud_storage/inventory/tests/inv_consumer_tests.cc

```diff
@@ -43,6 +43,8 @@ TEST(Consumer, ParseNTPFromPath) {
     std::vector<p> test_data{
       {"a0a6eeb8/kafka/topic-x/999_24/178-188-1574137-1-v1.log.1",
        std::make_optional(make_ntp("kafka", "topic-x", 999))},
+      {"d10492a6-2408-418e-9b6b-051697c5255b/k/t/1_24/---",
```

Comment:
is this how the real path looks? i suppose this is the cluster uuid? don't we still include the hash prefix too?

Reply:
Yes, I discovered similar paths while running ducktape test:

```
d10492a6-2408-418e-9b6b-051697c5255b/kafka/topic-kppbtczbbq/2_22/429-500-1153657-1-v1.log.1
```

and looking at the code leads to https://github.com/redpanda-data/redpanda/blob/dev/src/v/cloud_storage/remote_path_provider.cc#L143 and https://github.com/redpanda-data/redpanda/blob/dev/src/v/cloud_storage/segment_path_utils.cc#L20

If there is a label it looks like we use the cluster uuid, otherwise a hash prefix